test/bind.sh

```diff
@@ -1,23 +1,39 @@
 #!/bin/bash -e
 
-. shared.sh
+INSTANCE_ID=$1
+BINDING_ID=$2
+PLAN_UUID="7f4a5e35e4af2beb70076e72fab0b7ff"
+SERVICE_UUID="dh-postgresql-apb-s964m"
+
+if [ "$INSTANCE_ID" = "" ]
+then
+  echo "Usage: $0 <instance uuid> <binding uuid>"
+  exit
+fi
+
+if [ "$BINDING_ID" = "" ]
+then
+  echo "Usage: $0 <instance uuid> <binding uuid>"
+  exit
+fi
 
-# mlab with params
-instanceUUID="8c9adf85-9221-4776-aa18-aae7b7acc436"
 req="{
-  \"plan_id\": \"$planUUID\",
-  \"service_id\": \"$serviceUUID\",
+  \"plan_id\": \"$PLAN_UUID\",
+  \"service_id\": \"$SERVICE_UUID\",
+  \"context\": \"blog-project\",
   \"app_guid\":\"\",
   \"bind_resource\":{},
-  \"parameters\": {
-    \"user\": \"acct_one\"
-  }
+  \"parameters\": {}
 }"
 
 curl \
-  -X PUT \
-  -H 'X-Broker-API-Version: 2.9' \
-  -H 'Content-Type: application/json' \
-  -d "$req" \
-  -v \
-  http://localhost:1338/v2/service_instances/$instanceUUID/service_bindings/$bindingUUID
+    -k \
+    -X PUT \
+    -H "Authorization: bearer $(oc whoami -t)" \
+    -H "Content-type: application/json" \
+    -H "Accept: application/json" \
+    -H "X-Broker-API-Originating-Identity: " \
```

Comment:
I think you're going to need to set auto_escalate=true for these scripts to work, I would also document that.

Reply:
added to the readme